lib/commands/test-init.ts

```diff
@@ -37,6 +37,25 @@ class TestInitCommand implements ICommand {
 				'save-dev': true,
 				optional: false,
 			});
+
+			let modulePath = path.join(projectDir, "node_modules", mod);
+			let modulePackageJsonPath = path.join(modulePath, "package.json");
+			let modulePackageJsonContent = this.$fs.readJson(modulePackageJsonPath);
+			let modulePeerDependencies = modulePackageJsonContent.peerDependencies || {};
+
+			for (let peerDependency in modulePeerDependencies) {
```

Comment:
I think a good practice when iterating an object's keys in JS is to check for own properties only. You can use lodash if you want to: http://lodash.com/docs/4.17.4#forOwn